cli/cli-client/src/main/java/org/infinispan/cli/connection/rest/RestConnection.java

```diff
@@ -610,6 +633,13 @@ public String describeCounter(String container, String counter) throws IOExcepti
       return parseBody(fetch(() -> client.counter(counter).configuration()), String.class);
    }
 
+   @Override
+   public String describeTask(String container, String taskName) throws IOException {
+      List<Map<String, Object>> list = parseBody(fetch(() -> client.tasks().list()), List.class);
+      Optional<Map<String, Object>> task = list.stream().filter(i -> taskName.equals(i.get("name"))).findFirst();
+      return task.get().toString();
```

Comment:
`task.get()` will throw a `NoSuchElementException` if the task doesn't exist. 
is that what you want? wouldn't it be better to return an empty string?

Reply:
Changed to throw a Messages.noSuchResource() exception